app/components/Search.js

```diff
@@ -94,21 +101,34 @@ class Search extends Component {
           </TouchableHighlight>
 
           <ScrollView>
-            { photographers 
-              ? photographers.map((photographer, i) => {
-                return (
-                  <View key={i}>
-                    <Text>Photographer: {photographer.name}</Text>
-                    <Text>Total Score: {photographer.score}</Text>
-                    <Text>Total Likes: {photographer.likes}</Text>
-                    <Text>Total Downloads: {photographer.downloads}</Text>
-                    <Text>Username: {photographer.username}</Text>
-                  </View>
-                )
-              }) 
-              : <View><Text>No Photographers in this area</Text></View>
-            }
-          </ScrollView>
+             { photographers
+               ? photographers.map((photographer, i) => {
+                 return (
+                   <View key={i}>
+                     <Text>Photographer: {photographer.name}</Text>
+                     <Text>Total Score: {photographer.score}</Text>
+                     <Text>Total Likes: {photographer.likes}</Text>
+                     <Text>Total Downloads: {photographer.downloads}</Text>
+                     <Text>Username: {photographer.username}</Text>
+                   </View>
+                 )
+               })
+               : <View><Text>No Photographers in this area</Text></View>
+             }
+           </ScrollView>
+
+          <View>
```

Comment:
Awesome. I will merge this what I have got and put this into the photographer component.